cmd/cloud_login.go

```diff
@@ -126,29 +108,7 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		if err != nil {
 			return err
 		}
-		email := vals["Email"]
-		password := vals["Password"]
-
-		client := cloudapi.NewClient(
-			c.globalState.Logger,
-			"",
-			consolidatedCurrentConfig.Host.String,
-			consts.Version,
-			consolidatedCurrentConfig.Timeout.TimeDuration())
-
-		var res *cloudapi.LoginResponse
-		res, err = client.Login(email, password)
```

Comment:
I think that would be nice to have a similar method but to check the correctness of the given token. 
Like a health endpoint, but with authentication. Wdyt? cc/ @oleiade @dgzlopes

PS: I think that's not supported yet, but would require some changes in Backend.

Reply:
I agree! It would be nice to validate the token. 

Can we do this by just trying to call one of the authenticated APIs? But yeah, a dedicated endpoint maybe would be better. I think you should involve someone from backend :eyes:

Reply:
We discussed internally with the Backend team, and they'll add support for the `/login` endpoint to also accept a `token` for auth (as an alternative for now, for backwards compatibility, and as the unique method support in the future).

However, as that will take some time, I guess we can move forward with this PR and add that in the next iteration (I'll create an issue to do not forget).

Reply:
Sure @codebien! The purpose of such check would be to make the resulting user experience better.

Now, when the user logs in with email & password, they get immediate feedback whether those credentials were correct or not, and the token saved is always valid, as it is returned by the API.

However, after these changes, the user won't log in, but would just set a token. So, if we don't add an extra check to verify whether the token is valid or not, the user won't receive such feedback until they try to run another command. While, in my opinion, would be much better to give them the feedback of whether the token was valid or not at time of running `k6 cloud login`.

Does that sound good to you? Do you have any other suggestion?

Thanks! 🙇🏻

Reply:
Gotcha, that sounds good to me and valuable! :tada:

Reply:
I would argue that this was the behaviour before for `-t` on the cli.

I am not against both that and this to do some checks, but I would not block this PR on endpoint to check this.

Reply:
> I would argue that this was the behaviour before for `-t` on the cli.

Yeah, that was my initial thought, but having being like this doesn't mean we cannot improve the user experience, imo. Plus, the use of token was arguably an alternative for the main login flow with standard credentials (although I know that in practice that's no longer the case), but now it will become the main (and unique?), and so I think there's more reason to keep the main login flow with a good user experience, as it used to be the case for email and password.

> I am not against both that and this to do some checks, but I would not block this PR on endpoint to check this.

Yeah, I agree! That's why said above:

```
However, as that will take some time, I guess we can move forward with this PR and add that in the next iteration (I'll create an issue to do not forget).
```